back/engines/commercial/user_custom_fields/app/controllers/user_custom_fields/web_api/v1/user_custom_fields_controller.rb

```diff
@@ -64,6 +64,7 @@ def update
     end
 
     def reorder
+      fix_reordering
```

Comment:
Is this still necessary?

Reply:
Hmm, kind of. In that it will fix any platforms that currently have the issue. But maybe I should move this to a rake task instead. Given that it will become dead code in time.

Reply:
Yes, my preference would be a rake task indeed. Otherwise, I feel like we should add this to every `reorder` endpoint we have, but this would make our code a bit more complex (if a developer wants to understand why it's there and if it could be removed at some point).

Reply:
We might already have a rake task related to this, but not 100% sure